uber/site_sections/registration.py

```diff
@@ -165,15 +164,9 @@ def qrcode_generator(self, data):
         this function.  Or, offload image generation to a dedicated microservice that replicates this functionality.
 
         """
-        if not c.QR_CODE_PASSWORD or not qr_cipher:
-            # Don't generate a 2D barcode if we aren't configured for encryption
-            return '2D barcode password not set, cannot generate images'
-
-        encrypted_data = c.EVENT_QR_ID + bytes.decode(binascii.hexlify(qr_cipher.encrypt(uuid.UUID(data).hex)))
-
         checkin_barcode = treepoem.generate_barcode(
             barcode_type='azteccode',
-            data=encrypted_data,
+            data=c.EVENT_QR_ID + str(data),
```

Comment:
Note that adding `c.EVENT_QR_ID` doesn't really help avoid collisions, because most of the places where there would be a collision are going to occur on the same host if they occur at all. I would rather remove it here, since it's not adding anything, makes the code longer, complicates parsing slightly, and may contribute to a false sense of security in the future.